utils/recordGamedb.ts

```diff
@@ -0,0 +1,47 @@
+import prisma from "~/utils/db";
+
+type GameStat = {
+  player_id: string;
+  size: number;
+  food_eaten: number;
+  kills: number;
+  deaths: number;
+};
+
+type recordGameArgs = {
```

Comment:
nit: имена типов данных обычно пишут с большой буквы

Общее правило (которое используют во многих языках). Общие/абстрактные сущности, описывающие категорию/класс мы пишем с большой буквы.

С большой:
- имена классов
- имена типов данных
- имена интерфейсов

С маленькой:
- имена переменных
- имена функций

```suggestion
type RecordGameArgs = {
```